### PR TITLE
FEAT: Added new extension to parametrize edb

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by svandenb-dev — DISMISSED** [review]
LGTM

File: doc/source/User_guide/extensions.rst [hunk 1]
```diff
@@ -40,6 +40,11 @@ Project extension apply to all extensions that are applicable for all AEDT appli
 
             Lear how to convert projects from 2022R2 to newer versions.
 
+   .. grid-item-card:: Parametrize Layout
+            :link: pyaedt_extensions_doc/hfss3dlayout/parametrize_edb
+            :link-type: doc
+
+            Lear how to parametrize a full aedb.
```
Comment (MaxJPRey) [inline doc/source/User_guide/extensions.rst:47]:
```suggestion
            Learn how to parameterize a full aedb.
```

File: doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+``aedb_path`` and ``design_name`` define the source aedb project.
+``parametrize_layers``, ``parametrize_materials``, ``parametrize_padstacks``, ``parametrize_traces``
+define which parts of the aedb has to be parametrized while the ``nets_filter`` defines which nets has to be included.
+``expansion_polygon_mm`` and ``expansion_void_mm`` define if and which value of expansion has to be applied on
+polygons and voids.
+``relative_parametric`` define if the parameters have to be considered as a delta of the original value or not.
+``project_name`` is the new project name.
+
+You can also launch the extension user interface from the terminal. An example can be found here:
```
Comment (gmalinve) [inline doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst:28]:
```suggestion
The extension user interface can also be launched from the terminal. An example can be found here:
```

File: doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+==================
+
+You can parametrize stackup, materials, padstacks and traces of an existing 3D Layout design and also
+change size of voids and polygons to conduct corner analysis.
+
+You can access the extension from the icon created on the **Automation** tab using the Extension Manager.
+
+The following image shows the extension user interface:
+
+.. image:: ../../../_static/extensions/parametrize.png
```
Comment (gmalinve) [inline doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst:11]: where is the image? I can't see it as an added file

**Review by gmalinve — CHANGES_REQUESTED** [review]

**Review by gkorompi — APPROVED** [review]
LGTM

File: doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst (hunk 3 above)
Comment (maxcapodi78) [reply doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst:11]: images have been pushed now.

File: doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+The available arguments are: ``aedb_path``, ``design_name``, ``parametrize_layers``,
+``parametrize_materials``, ``parametrize_padstacks``, ``parametrize_traces``, ``nets_filter``,
+``expansion_polygon_mm``, ``expansion_void_mm``, ``relative_parametric``, ``project_name``.
+
+``aedb_path`` and ``design_name`` define the source aedb project.
+``parametrize_layers``, ``parametrize_materials``, ``parametrize_padstacks``, ``parametrize_traces``
+define which part of the aedb has to be parametrized while the ``nets_filter`` defines which net has to be included.
+``expansion_polygon_mm`` and ``expansion_void_mm`` define if and which value of expansion has to be applied on
+polygons and voids.
+``relative_parametric`` define if the parameters have to be considered as a delta of the original value or not.
```
Comment (gmalinve) [inline doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst:25]:
```suggestion
``relative_parametric`` defines if the parameters have to be considered as a delta of the original value.
```

**Review by gmalinve — APPROVED** [review]

**Review by gmalinve — APPROVED** [review]

**Pull request merged**